### PR TITLE
Set verbose deafult to 0 for test/mpi/errors/allgatherlength.c and reduceop.c

### PR DESCRIPTION
**Pull request by Jayyee-HPC**
## Pull Request Description
Set verbose default to 0 for test/mpi/errors/allgatherlength.c and reduceop.c

The default value of verbose is 1 in these two files, which means even if the errors are caught, the test script will mark these tests failed because of unexpected output. 

`An example
  ---
not ok  - ./errors/coll/reduceop 4 # time=0.0381298065185547
  Directory: ./errors/coll
  File: reduceop
  Num-procs: 4
  Timeout: 180
  Date: "Mon May  3 22:49:31 2021"
  ...
Test output (expected 'No Errors'):
Found expected error; message is: Invalid argument, error stack:
MPIR_Coll_len_check_scatter(367): message sizes do not match across processes in the collective routine
Result : 
1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 
Found expected error; message is: Invalid argument, error stack:
MPIR_Coll_len_check_scatter(367): message sizes do not match across processes in the collective routine
Found expected error; message is: Invalid argument, error stack:
MPIR_Coll_len_check_scatter(367): message sizes do not match across processes in the collective routine
Found expected error; message is: Invalid argument, error stack:
MPIR_Coll_len_check_scatter(367): message sizes do not match across processes in the collective routine
No Errors`

## Author Checklist
* [x] **Provide Description** 
      Particularly focus on _why_, not _what_. Reference background, issues, test failures, xfail entries, etc.
* [x] **Commits Follow Good Practice**
      Commits are self-contained and do not do two things at once. 
      Commit message is of the form: `module: short description` 
      Commit message explains what's in the commit.
* [x] **Passes All Tests**
      Whitespace checker. Warnings test. Additional tests via comments.
* [x] **Contribution Agreement**
      For non-Argonne authors, check [contribution agreement](http://www.mpich.org/documentation/contributor-docs/). 
      If necessary, request an explicit comment from your companies PR approval manager.

### CODE REVIEW

**Comment by hzhou** [comment]
@Jayyee-HPC Both commits are the same type of fix and will always be reviewed or picked together. Could you squash them into a single commit?

**Comment by Jayyee-HPC** [comment]
> @Jayyee-HPC Both commits are the same type of fix and will always be reviewed or picked together. Could you squash them into a single commit?

The commits were squashed into one.

**Review by hzhou — APPROVED** [review]
Looks good. Will merge after tests cleared.

**Comment by hzhou** [comment]
@Jayyee-HPC Can you do a quick rebase?

**Comment by hzhou** [comment]
@Jayyee-HPC Do not merge `main` back into the PR branch. Update your local `main` branch, then `git rebase main`.

**Comment by Jayyee-HPC** [comment]
> @Jayyee-HPC Do not merge `main` back into the PR branch. Update your local `main` branch, then `git rebase main`.

I did `git rebase upstream/main`; upstream links to pmodels/mpich

**Comment by Jayyee-HPC** [comment]
@hzhou I used the main branch to submit this PR. Is this the issue?

**Comment by hzhou** [comment]
> 
> 
> @hzhou I used the main branch to submit this PR. Is this the issue?

It still can work but more difficult to track. We usually leave the main branch tracking upstream and do PRs out-of new branches.

**Comment by Jayyee-HPC** [comment]
> > @hzhou I used the main branch to submit this PR. Is this the issue?
> 
> It still can work but more difficult to track. We usually leave the main branch tracking upstream and do PRs out-of new branches.

I see. Do I need to re-submit a PR? This check takes suspicious long.

**Comment by hzhou** [comment]
test:mpich/warnings

**Comment by hzhou** [comment]
> 
> 
> > > @hzhou I used the main branch to submit this PR. Is this the issue?
> > 
> > 
> > It still can work but more difficult to track. We usually leave the main branch tracking upstream and do PRs out-of new branches.
> 
> I see. Do I need to re-submit a PR? This check takes suspicious long.

No. It looks fine now

**Comment by hzhou** [comment]
test:mpich/warnings

**Comment by hzhou** [comment]
test:mpich/warnings

**Pull request merged**
